ts/sagas/startup/loadServiceRequestHandler.ts

```diff
@@ -28,6 +39,28 @@ export function* loadServiceRequestHandler(
     }
 
     if (response.value.status === 200) {
+      // If the organization fiscal code is associated to different organizazion names,
+      // it is considered valid the one declared for a visible service
+      const visibleServices: VisibleServicesState = yield select(
+        visibleServicesSelector
+      );
+      const service = pot.some(response.value.value);
+      const isVisible = isVisibleService(visibleServices, service) || false;
+
+      const organizations: OrganizationNamesByFiscalCodeState = yield select(
+        organizationNamesByFiscalCodeSelector
+      );
+      const fc = service.value.organization_fiscal_code;
+
+      if (organizations) {
+        const organization = organizations[fc];
+        // If the organization has been previously saved in the organization entity,
+        // the organization name  is updated only if the related service is visible
+        if (!organization || (organization && isVisible)) {
```

Comment:
```suggestion
        if (!organization || (organization &&
            isVisible &&
            organization !== response.value.value.organization_name)) {
```
Does this further control make sense?

Reply:
Looking both at `backendClient.getService` and at the `GetServiceT` definition, if the response is 200,  the response content should have type `ServicePublic`: from the type definition the `organization_name` attribute is set as required, so I expected it to be always included into the response.

Reply:
@valentinacala yes you are right. I just suggested a check to verify if the organization names are different (the stored one with the just loaded) `organization !== response.value.value.organization_name` The goal is updated with the new one if they different, I guess.
Maybe I'm missing somethings and this check doesn't make sense

Reply:
You're right, I confused it with a different kind of check. Updated within  https://github.com/teamdigitale/io-app/pull/1328/commits/7c1350a8fd726d0f5d4fb88ff81e0df88015c51c